### PR TITLE
feat: studio (#104)

### PR DESCRIPTION
**Pull request by MaloPolese**
Closes # 

## ✅ Checks

- [ ] My pull request adheres to the code style of this project
- [ ] My code is documented
- [ ] I provided unitary tests or procedure to test my code
- [ ] My PR have a clear description of what my code is supposed to do

### CODE REVIEW

File: .github/workflows/package.yml [hunk 1]
```diff
@@ -9,7 +9,18 @@ on:
 
 jobs:
   package:
+    name: Build
     runs-on: ubuntu-22.04
+    strategy:
+      matrix:
+        target:
+          - polyflix
+          - cmd
```
Comment (kalil-pelissier) [inline .github/workflows/package.yml:18]: What is cmd ?

File: .github/workflows/package.yml [hunk 2]
```diff
@@ -9,7 +9,18 @@ on:
...
+        include:
+          - target: polyflix
+            docker-file: apps/polyflix/Dockerfile
+          - target: cmd
```
Comment (kalil-pelissier) [inline .github/workflows/package.yml:22]: Same

**Review by kalil-pelissier — CHANGES_REQUESTED** [review]

File: .github/workflows/package.yml [hunk 3]
```diff
@@ -39,10 +50,11 @@ jobs:
           build-args: |
             BUILD_MODE=${{ steps.extract-env.outputs.environment }}
           context: .
-          file: Dockerfile.app
+          file: ${{ matrix.docker-file }}
           push: true
           tags: ${{ steps.meta.outputs.tags }}
```
Comment (kalil-pelissier) [inline .github/workflows/package.yml:55]: Add a spécifique tag to distinguish cms and frontend:

```
 uses: docker/metadata-action@v4
        with:
          images: ghcr.io/frontend
          flavor: suffix=-migrations
```

**Review by kalil-pelissier — CHANGES_REQUESTED** [review]

File: .github/workflows/package.yml [hunk 4]
```diff
@@ -21,7 +33,9 @@ jobs:
         uses: docker/metadata-action@v4
         with:
           images: ghcr.io/${{ github.repository }}
-          flavor: ${{ inputs.docker-metadata-flavor }}
+          flavor: |
+            ${{ inputs.docker-metadata-flavor }}
```
Comment (kalil-pelissier) [inline .github/workflows/package.yml:37]: Remove this ligne

**Review by kalil-pelissier — CHANGES_REQUESTED** [review]

**Review by kalil-pelissier — APPROVED** [review]

**Review by thomasgouveia — APPROVED** [review]

**Pull request merged**